### DISCUSSION TITLE
Steno option to capture the first x bytes of each connection

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#2394] [Unsupported Versions] [answered]
Right now, I'm using BPF for SMB and SSL to restrict the size of nsm. However, BPF is only an on/off switch. 
Could it be possible to specify in Steno the amount of first bytes to capture for each connection ? This way, the connection initialization could still be captured providing useful metadata while excluding the large data exchange.

**Comment by dougburks** [comment, marked as answer]
There is an open issue for this over on the Stenographer repo:
https://github.com/google/stenographer/issues/123

**Reply by 0nnyx** [reply]
Thx for the response Doug. Doesn't look to be coming given this feature request was raised in 2015
